modules/querying/pages/func/query-user-defined-functions.adoc

```diff
@@ -160,20 +154,30 @@ namespace UDIMPL {
 
   // other UDFs ...
```

Comment:
Did you see all of this in the UDF file you downloaded? What version of TigerGraph were you using?